### ISSUE TITLE
no output for the reaction

### ISSUE THREAD

**Issue by autodataming** [asad/ReactionDecoder#14]
```
java -jar ReactionDecoder.jar -Q SMI -q "BrB(Br)Br.O=C(COCc1ccccc1)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21.O>>O=C(CO)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21" -g -j AAM -f TEXT
```
no output for the reaction

**Comment by autodataming** [comment]
```
java -jar ReactionDecoder.jar -Q SMI -q "BrB(Br)Br.O=C(COCc1ccccc1)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21.O>>O=C(CO)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21" -g -j AAM -f TEXT
```

```
0    [IJava-executor-0] INFO  net.sf.jnati.deploy.artefact.ConfigManager  - Loading global configuration
9    [IJava-executor-0] DEBUG net.sf.jnati.deploy.artefact.ConfigManager  - Loading defaults: jar:file:/home/xxx/jupyternotebook/./rdt.jar!/META-INF/jnati/jnati.default-properties
10   [IJava-executor-0] INFO  net.sf.jnati.deploy.artefact.ConfigManager  - Loading artefact configuration: jniinchi-1.03_1
12   [IJava-executor-0] DEBUG net.sf.jnati.deploy.artefact.ConfigManager  - Loading instance defaults: jar:file:/home/xxx/jupyternotebook/./rdt.jar!/META-INF/jnati/jnati.instance.default-properties
15   [IJava-executor-0] INFO  net.sf.jnati.deploy.repository.ClasspathRepository  - Searching classpath for: jniinchi-1.03_1-LINUX-AMD64
19   [IJava-executor-0] INFO  net.sf.jnati.deploy.repository.LocalRepository  - Searching local repository for: jniinchi-1.03_1-LINUX-AMD64
19   [IJava-executor-0] DEBUG net.sf.jnati.deploy.repository.LocalRepository  - Artefact path: /home/xxx/.jnati/repo/jniinchi/1.03_1/LINUX-AMD64
21   [IJava-executor-0] INFO  net.sf.jnati.deploy.artefact.ManifestReader  - Reading manifest
71   [IJava-executor-0] INFO  net.sf.jnati.deploy.NativeArtefactLocator  - Artefact (jniinchi-1.03_1-LINUX-AMD64) location: /home/xxx/.jnati/repo/jniinchi/1.03_1/LINUX-AMD64
71   [IJava-executor-0] DEBUG net.sf.jnati.deploy.NativeLibraryLoader  - Loading library: /home/xxx/.jnati/repo/jniinchi/1.03_1/LINUX-AMD64/libJniInchi-1.03_1-LINUX-AMD64.so
```

**Comment by autodataming** [comment]
NO OUTPUT FOR THE REACTION

```
java -jar ReactionDecoder.jar -Q SMI -q "BrB(Br)Br.O=[N+]([O-])c1cnc2[nH]nc(COCc3ccccc3)c2c1.O=C([O-])O.CCOC(C)=O>>O=[N+]([O-])c1cnc2[nH]nc(CO)c2c1" -g -j AAM -f TEXT

```

**Comment by autodataming** [comment]
NO OUTPUT FOR THE REACTION

```
'Brc1ncc(C2CC2)cn1.COC(=O)C1C(c2ccccc2)C1c1ccc(Br)cc1>>COC(=O)C1C(c2ccccc2)C1c1ccc(-c2ncc(C3CC3)cn2)cc1'
```

**Comment by autodataming** [comment]
NO OUTPUT FOR THE REACTION

```
"O=C(COCc1ccccc1)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21>>O=C(CO)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21" 
```



java -jar /home/zgong/nfs/czq/workflow_retro_deepsyn2/step1data/USPTO1976_2016/rdtmap/ReactionDecoder.jar -Q SMI -q "O=C(COCc1ccccc1)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21>>O=C(CO)N1CCCc2sc(-c3ccc(O[C@@H]4C[C@@H](N5CCCCC5)C4)cc3)nc21" -g -j AAM -f TEXT


!--------------------------------------------------------
Reaction Decoder Tool (RDT)

Author: Syed Asad Rahman
e-mail: asad@ebi.ac.uk|s9asad@gmail.com
c/o EMBL-European BioInformatics Institute (EBI)
WTGC, CB10 1SD Hinxton
UK

Note: The copyright of this software belongs to the author
and EMBL-European BioInformatics Institute (EBI).

Reference
Rahman, S.A. et.al.(2016) Reaction Decoder Tool (RDT):
Extracting Features from Chemical Reactions, Bioinformatics (2016)
doi: 10.1093/bioinformatics/btw096
!--------------------------------------------------------

-- AAM --
0    [main] INFO  net.sf.jnati.deploy.artefact.ConfigManager  - Loading global configuration
3    [main] DEBUG net.sf.jnati.deploy.artefact.ConfigManager  - Loading defaults: jar:file:/home/NFS/user/zgong/czq/workflow_retro_deepsyn2/step1data/USPTO1976_2016/rdtmap/ReactionDecoder.jar!/META-INF/jnati/jnati.default-properties
3    [main] INFO  net.sf.jnati.deploy.artefact.ConfigManager  - Loading artefact configuration: jniinchi-1.03_1
4    [main] DEBUG net.sf.jnati.deploy.artefact.ConfigManager  - Loading instance defaults: jar:file:/home/NFS/user/zgong/czq/workflow_retro_deepsyn2/step1data/USPTO1976_2016/rdtmap/ReactionDecoder.jar!/META-INF/jnati/jnati.instance.default-properties
5    [main] INFO  net.sf.jnati.deploy.repository.ClasspathRepository  - Searching classpath for: jniinchi-1.03_1-LINUX-AMD64
5    [main] INFO  net.sf.jnati.deploy.repository.LocalRepository  - Searching local repository for: jniinchi-1.03_1-LINUX-AMD64
6    [main] DEBUG net.sf.jnati.deploy.repository.LocalRepository  - Artefact path: /home/zgong/.jnati/repo/jniinchi/1.03_1/LINUX-AMD64
6    [main] INFO  net.sf.jnati.deploy.artefact.ManifestReader  - Reading manifest
24   [main] INFO  net.sf.jnati.deploy.NativeArtefactLocator  - Artefact (jniinchi-1.03_1-LINUX-AMD64) location: /home/zgong/.jnati/repo/jniinchi/1.03_1/LINUX-AMD64
24   [main] DEBUG net.sf.jnati.deploy.NativeLibraryLoader  - Loading library: /home/zgong/.jnati/repo/jniinchi/1.03_1/LINUX-AMD64/libJniInchi-1.03_1-LINUX-AMD64.so

**Comment by asad** [comment]
Please check your input command.

java -jar ReactionDecoder.jar -Q SMI -q "O=C(COCc1ccccc1)N1CCCc2sc(-c3ccc(O[C@@h]4CC@@HC4)cc3)nc21"  -T SMI -t "O=C(CO)N1CCCc2sc(-c3ccc(O[C@@h]4CC@@HC4)cc3)nc21" -g -j AAM -f TEXT

**Comment by asad** [comment]
Thank you. This has been resolved in version 2.3.
Also please check your reaction SMILES.
